### PR TITLE
[NONMOD] Locks tribal weapons to ashwalkers

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request

~~removes the bone sword crafting recipe~~ Reverted
Removes all tribal weapons from normal crafting, enable only ashwalkers to use them
Rebalances the bone spear.
The ash village has a bone sword now in its map (I couldn't be bothered to remove it after redoing the PR)

## How This Contributes To The Skyrat Roleplay Experience

Frankly: It isn't balanced for the station.
~~The Bone Sword doesn't fit any design space for tribal weapons. It's too strong for a one-handed weapon, but there's already good 2-hand options for ashwalkers and the bone dagger is the best one handed option you should be able to make from just bones and stuff. It's also absurdly easy to craft for how powerful it is. Given public mining exists (miners themselves have decent weapon options already) it's way too easy to get these and bring them up.~~

I've gone the extra mile and just locked all tribal weapons to ashwalkers. They are insanely easy to craft for how insanely strong they are, and public mining makes it too easy to get the materials. I'm leaving tribal weapon balance to ashwalkers, they can fight over numbers themselves.

The spear was, however, extremely strong for how safe of a weapon option it is on Skyrat. Not only is it 20(!!) force when dual handed but it also has reach AND armour pen. You can't have everything and still have it mass-producible for the price of a watcher or two. It's been knocked down a peg.

Other weapon options the station AND ashwalkers can acquire are being filled design-wise by Forging, which I am rebalancing in another PR.

## Changelog

:cl:
balance: Tribal weapons are now ashwalker-only
balance: Bone spear was nerfed.
qol: adds a bone sword to the ash walker village.
/:cl:

### CODE REVIEW

**Comment by OzzyMcPseudonym** [comment]
Hey, I have some constructive feedback on how this could be improved. While I agree that bone swords absolutely should not be attainable by station crew, perhaps it would be within reason to make them an ashwalker-only crafting recipe similar to how the bone keys can only be crafted by ashies?

File: code/datums/components/crafting/recipes.dm [hunk 1]
```diff
@@ -927,6 +927,7 @@
 	result = /obj/item/knife/combat/bone
 	time = 20
 	reqs = list(/obj/item/stack/sheet/bone = 2)
+	always_available = FALSE
```
Comment (jjpark-kb) [inline code/datums/components/crafting/recipes.dm]: skyrat edit comment

File: code/datums/components/crafting/recipes.dm [hunk 2]
```diff
@@ -935,6 +936,7 @@
 	time = 30
 	reqs = list(/obj/item/stack/sheet/bone = 4,
 				/obj/item/stack/sheet/sinew = 1)
+	always_available = FALSE
```
Comment (jjpark-kb) [inline code/datums/components/crafting/recipes.dm]: skyrat edit comment

File: code/datums/components/crafting/recipes.dm [hunk 3]
```diff
@@ -943,6 +945,7 @@
 	time = 50
 	reqs = list(/obj/item/stack/sheet/bone = 6,
 				/obj/item/stack/sheet/sinew = 3)
+	always_available = FALSE
```
Comment (jjpark-kb) [inline code/datums/components/crafting/recipes.dm]: skyrat edit comment

File: code/game/objects/items/spear.dm [hunk 4]
```diff
@@ -179,14 +179,14 @@
...
-	force = 12
+	force = 10 //SKYRAT EDIT
 	throwforce = 22
 	reach = 2 // SKYRAT EDIT
 	armour_penetration = 15 //Enhanced armor piercing
 
 /obj/item/spear/bonespear/ComponentInitialize()
 	. = ..()
-	AddComponent(/datum/component/two_handed, force_unwielded=12, force_wielded=20, icon_wielded="[icon_prefix]1")
+	AddComponent(/datum/component/two_handed, force_unwielded=10, force_wielded=16, icon_wielded="[icon_prefix]1") //SKYRAT EDIT
```
Comment (jjpark-kb) [inline code/game/objects/items/spear.dm]: honestly would make it 8 to 16

File: code/modules/antagonists/ashwalker/ashwalker.dm [hunk 5]
```diff
@@ -33,6 +33,11 @@
 	RegisterSignal(owner.current, COMSIG_MOB_EXAMINATE, .proc/on_examinate)
 	owner.teach_crafting_recipe(/datum/crafting_recipe/skeleton_key)
 	owner.teach_crafting_recipe(/datum/crafting_recipe/ashnecklace) //SKYRAT EDIT DRACONIC NECKLACE//
+	owner.teach_crafting_recipe(/datum/crafting_recipe/bonesword)	//SKYRAT EDIT TRIBAL WEAPONS FOR TRIBALS//
+	owner.teach_crafting_recipe(/datum/crafting_recipe/macahuitl)	//SKYRAT EDIT
+	owner.teach_crafting_recipe(/datum/crafting_recipe/boneaxe)		//SKYRAT EDIT
+	owner.teach_crafting_recipe(/datum/crafting_recipe/bonespear)	//SKYRAT EDIT
+	owner.teach_crafting_recipe(/datum/crafting_recipe/bonedagger)	//SKYRAT EDIT
```
Comment (jjpark-kb) [inline code/modules/antagonists/ashwalker/ashwalker.dm:40]: we really should find an alternative way to doing this... not saying to do it for this PR

**Pull request merged**